### PR TITLE
Add libdbus-1-dev and updated git to docker image

### LINKED ISSUE #1463: add libdbus-1-dev to the Docker image
#### Problem
Need this for https://github.com/project-chip/connectedhomeip/pull/1438

### PR DESCRIPTION
**Pull request by andy31415**
#### Problem
We need more things in the docker file

 #### Summary of Changes
Adds libdbus-1-dev to the image, updates the git version (github actions will do real git checkouts) and adds pygit to the python packages that are supported.

Version bump by 2 because changes made in https://github.com/project-chip/connectedhomeip/pull/1441 are already active as version 17.

fixes #1463 1463

### CODE REVIEW

**Comment by andy31415** [comment]
curl and gettext are for git compilation

**Review by mspang — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

File: integrations/docker/images/chip-build/Dockerfile [hunk 1]
```diff
@@ -60,6 +62,19 @@ RUN set -x \
     && rm cmake-3.17.2.tar.gz \
     && : # last line
 
+# Install newer git version, to allow for github workflow checkout
+RUN set -x \
```
Comment (rwalker-apple) [inline integrations/docker/images/chip-build/Dockerfile:66]: I was just updating us to ubuntu:focal, which already has this and an up-to-date cmake

Comment (andy31415) [reply integrations/docker/images/chip-build/Dockerfile:66]: Is that change already in?

I have already built the 0.2.18 image in dockerhub to unblock linux with libdbus-1-dev availability. I am afraid that if we keep the PRs the images and sources become out of sync.

**Comment by andy31415** [comment]
#1508 makes this one obsolete.

**Pull request closed without merging**